server/.golangci.yml

```diff
@@ -97,7 +97,6 @@ issues:
         channels/app/platform/helper_test.go|\
         channels/app/platform/license.go|\
         channels/app/platform/session.go|\
-        channels/app/platform/status.go|\
```

Comment:
just for my own understanding -- does this mean we _were_ checking `status.go` but now we're not?

Reply:
No, this is an ignore list. We were NOT checking before. Now we are.